src/commons/BrowserUtils.ts

```diff
@@ -774,6 +793,18 @@ export namespace BrowserUtils {
     }
   }
 
+  /**
+   *
+   * @param selector - element for get size
+   */
+  export function getElementSize(selector: string): SizeReturn {
+    Reporter.debug(`Get Element: '${selector}' size`);
+    isExist(selector);
+    const element: Element<void> = $(selector);
+
+    return element.getSize();
```

Comment:
as we talked, you may use $(selector).getSize() direfctly

Reply:
done